src/sage/numerical/optimize.py

```diff
@@ -519,9 +519,9 @@ def minimize_constrained(func, cons, x0, gradient=None, algorithm='default', **a
         sage: x, y = var('x y')
         sage: f(x,y) = (100 - x) + (1000 - y)
         sage: c(x,y) = x + y - 479  # > 0
-        sage: minimize_constrained(f, [c], [100, 300])
+        sage: minimize_constrained(f, [c], [100, 300]) # random
         (805.985..., 1005.985...)
-        sage: minimize_constrained(f, c, [100, 300])
+        sage: minimize_constrained(f, c, [100, 300])   # random
```

Comment:
just remove the whole test then? There's no value if nobody sees it and it isn't tested right?

That said, is there a way to test that the issue is indeed fixed?

Reply:
Yes, `# random` does report errors, as opposed to `# not tested`.